gas/constraints.lisp

```diff
@@ -103,7 +103,7 @@
 ;; though: it still asserts "smallness" so that it
 ;; should filter out MXPX induced out of gas exceptions.
 (defconstraint asserting-the-gas-cost-is-nonnegative (:guard FIRST)
-  (call-to-LT 1        ;; row shift parameter
+  (call-to-LEQ 1       ;; row shift parameter
```

Comment:
😁 Was that the root cause of all the tests blowing up ?

Reply:
They are still running, but it seems so.